miasm2/core/graph.py

```diff
@@ -162,42 +148,113 @@ def dot(self):
         return out
 
 
+    def reachable_sons(self, head):
```

Comment:
To factorize and avoid future copy/paste error on patches, you could use a protected method with a callback (`successors_iter` for `reachable_sons` and `predecessors_iter` for `reachable_parents`)